shape.h

```diff
@@ -7,7 +7,7 @@ class Shape
 	{
 		m_shape = x;
 	}
-	bool getShape();
+	std::string getShape();
 	
 	char m_shape;
```

Comment:
Good. Now m_shape being instead of type char, make it of type EShape where EShape is an enum.